can-stache-bindings.js

```diff
@@ -349,6 +349,31 @@ var behaviors = {
 		var completedData = behaviors.initializeViewModel(dataBindings, initialViewModelData, function(){
 			// we need to make sure we have the viewModel available
 			bindingContext.viewModel = makeViewModel.apply(this, arguments);
+
+			// Here we want to do a dev-mode check to see whether the child does type conversions on
+			//  any two-way bindings.  This will be ignored and the child and parent will be desynched.
+			dataBindings.forEach(function(dataBinding) {
```

Comment:
remove in production

Reply:
Using the can-bind warning now, which is removed in production.